src/AeroSharp/AeroSharp.csproj

```diff
@@ -1,12 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>0.3.1</Version>
+    <Version>1.0.0</Version>
```

Comment:
I'd imagine we put something like `6.0.0` here although not sure how to manage the `7.0.0` for `net7.0` without introducing a nuspec

Reply:
I do want to at least mark this release as a major version upgrade, so I agree with you, but I don't want to necessarily complicate things too much here with versioning.  

Happy to maybe then just upgrade to 6.0.0 for now and then when we switch over to just .net7, that can be the next major upgrade version number?

Reply:
I think the problem with binding the version to the framework version is that it means we're effectively choosing to either:
- Not follow SemVer (bad)
- Not release breaking changes to the library except on the release cadence of the framework or the aerospike client version (bad)

I agree that:
- This change should be a major version change

Reply:
Would folks then be good here (I don't want to prematurely close a conversation) to jump to 1.0.0 as the first major version and then for future releases, we base off of this major version upgrade?  

We're trying to version this Aerosharp client, not relate it to anything that it's based on.  I think it was just coincidence, imo, that Aerospike's client got to v6 at the same time that they did a .net6 target.

Reply:
I just probably didn't quite get the original intention for targeting multiple sdks. In that case `6.0.0` should work just fine

Reply:
> Would folks then be good here (I don't want to prematurely close a conversation) to jump to 1.0.0 as the first major version and then for future releases, we base off of this major version upgrade?

This makes sense to me.

> We're trying to version this Aerosharp client, not relate it to anything that it's based on. I think it was just coincidence, imo, that Aerospike's client got to v6 at the same time that they did a .net6 target.

This interpretation also makes sense to me.

Reply:
Chatted IRL with @adenisov and we're gonna go ahead with 1.0.0 for now!

Reply:
+1 to bumping this to just 1.0.0. 

I'm not a fan of binding the library version to the client version due to the issues that @farmerau mentioned already. 

Reiterating here that we should follow SemVer and freely release new major versions (although we should always keep a keen eye on that to avoid fatiguing our users with breaking changes).